src/lib/queryString/queryString.ts

```diff
@@ -0,0 +1,39 @@
+import qs, { ParsedQs } from "qs"
+import { navigate } from "@reach/router"
+
+export const parse = (search: string) => qs.parse(search, { ignoreQueryPrefix: true })
+const stringify = (parsed: ParsedQs) => qs.stringify(parsed, { addQueryPrefix: true })
+
+const removeProp = (obj: object, prop: string) => Object
```

Comment:
It's unclear to me what `obj: object` is here. Can we specify the type more? Maybe something like `type QueryParams = Record<string, string | number>`?